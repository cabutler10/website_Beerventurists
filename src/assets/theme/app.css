html {
  height: 100%;
}

body {
  margin: 0;
  padding: 0;
  font-family: 'Open Sans', sans-serif;
  background-color: #ffffff;
  overflow-x: hidden;
  font-size: 14px;
  width: '100%';
}

a {
  text-decoration: none;
}
