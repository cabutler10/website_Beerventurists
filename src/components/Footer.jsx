import React from 'react'
import Link from './Link'
import { useTranslation } from 'react-i18next'
import { makeStyles } from '@material-ui/styles'
import Typography from '@material-ui/core/Typography'
import Button from '@material-ui/core/Button'
import classnames from 'classnames'
import InstagramIcon from '../assets/icons/Instagram'
import FacebookIcon from '../assets/icons/Facebook'
import Contact from './Contact'
import Newsletter from './Newsletter'

const useStyles = makeStyles(theme => ({
  root: {
    backgroundColor: theme.status.black,
    width: '100%',
  },
  container: {
    display: 'flex',
    flexWrap: 'wrap',
    justifyContent: 'space-between',
    padding: 15,
    [theme.breakpoints.up('sm')]: {
      paddingTop: 30,
      paddingBottom: 30,
      paddingLeft: 60,
      paddingRight: 60,
    },
    [theme.breakpoints.up('md')]: {
      paddingTop: 45,
      paddingBottom: 45,
      paddingLeft: 170,
      paddingRight: 170,
    },
    [theme.breakpoints.up('lg')]: {
      paddingTop: 45,
      paddingBottom: 45,
      paddingLeft: 170,
      paddingRight: 170,
    },
    [theme.breakpoints.up('xl')]: {
      paddingTop: 90,
      paddingBottom: 90,
      paddingLeft: 400,
      paddingRight: 400,
    },
  },
  containerInner: {
    display: 'flex',
    height: '100%',
  },
  flexItem: {
    flexBasis: '100%',
    paddingTop: 15,
    [theme.breakpoints.up('md')]: {
      flexBasis: '45%',
      paddingTop: 0,
    },
  },
  flexItemInner: {
    flexBasis: '50%',
    display: 'flex',
    flexDirection: 'column',
    justifyContent: 'space-between',
  },
  text: {
    color: theme.palette.common.white,
    fontSize: 12,
    textTransform: 'capitalize',
    paddingBottom: 10,
    [theme.breakpoints.up('sm')]: {
      lineHeight: 2,
    },
  },
  textLink: {
    '&:hover': {
      color: theme.palette.primary.main,
    },
  },
  textHeading: {
    color: theme.palette.common.white,
    fontSize: 12,
    textTransform: 'uppercase',
    paddingBottom: 15,
    paddingTop: 15,
    textAlign: 'center',
    [theme.breakpoints.up('md')]: {
      textAlign: 'left',
      paddingTop: 0,
    },
  },
  button: {
    borderColor: theme.status.white,
    borderRadius: 0,
    minWidth: 0,
    padding: 8,
    marginRight: 30,
    '&:hover, &:focus': {
      borderColor: theme.palette.primary.main,
      '& $icon': {
        color: theme.palette.primary.main,
      },
    },
    [theme.breakpoints.up('sm')]: {
      marginRight: 15,
    },
  },
  icon: {
    width: 18,
    height: 18,
    color: theme.palette.common.white,
  },
  iconContainer: {
    display: 'flex',
    justifyContent: 'flex-end',
  },
  link: {
    color: theme.palette.common.white,
    border: 'none',
    background: 'transparent',
    textDecoration: 'none',
    cursor: 'pointer',
    paddingLeft: 5,
    paddingRight: 5,
    '&:hover': {
      color: theme.palette.primary.main,
    },
  },
  legal: {
    backgroundColor: theme.status.black,
  },
  copyright: {
    fontSize: 12,
    textTransform: 'capitalize',
    color: theme.palette.common.white,
    letterSpacing: 3,
    paddingTop: 15,
    paddingBottom: 15,
  },
}))

function Footer() {
  const classes = useStyles()
  const { t } = useTranslation()

  return (
    <>
      <div className={classes.root}>
        <div className={classes.container}>
          <div className={classes.flexItem}>
            <Contact />
          </div>
          <div className={classes.flexItem}>
            <div className={classes.containerInner}>
              <div className={classes.flexItemInner}>
                <div>
                  <Typography className={classes.textHeading}>
                    {t('footer.links')}
                  </Typography>
                  <Link to="/returns/">
                    <Typography
                      className={classnames(classes.text, classes.textLink)}
                    >
                      {t('header.breweries')}
                    </Typography>
                  </Link>
                  <Link to="/payment/">
                    <Typography
                      className={classnames(classes.text, classes.textLink)}
                    >
                      {t('header.bars')}
                    </Typography>
                  </Link>
                  <Link to="/shipping/">
                    <Typography
                      className={classnames(classes.text, classes.textLink)}
                    >
                      {t('header.shops')}
                    </Typography>
                  </Link>
                  <Link to="/returns/">
                    <Typography
                      className={classnames(classes.text, classes.textLink)}
                    >
                      {t('header.guides')}
                    </Typography>
                  </Link>
                </div>
              </div>
              <div className={classes.flexItemInner}>
                <div>
                  <Typography className={classes.textHeading}>
                    {t('common.subscribe')}
                  </Typography>
                  <Typography className={classes.text}>
                    {t('common.subscribeText')}
                  </Typography>
                  <Newsletter variant="small" />
                </div>
                <div className={classes.iconContainer}>
                  <Button
                    variant="outlined"
                    className={classes.button}
                    href="https://www.facebook.com/profile.php?id=100016383813311"
                    target="_blank"
                    rel="noopener"
                  >
                    <FacebookIcon className={classes.icon} />
                  </Button>
                  <Button
                    variant="outlined"
                    className={classes.button}
                    href="https://www.instagram.com/beerventurists/"
                    target="_blank"
                    rel="noopener"
                  >
                    <InstagramIcon className={classes.icon} />
                  </Button>
                </div>
              </div>
            </div>
          </div>
        </div>
        <div className={classes.legal}>
          <Typography
            variant="body2"
            align="center"
            className={classes.copyright}
          >
            &copy; BEERVENTURISTS
            {/* |{' '}
            <Link to="/legal/" className={classes.link}>
              {t('footer.imprint')}
            </Link>
            |{' '}
            <Link to="/privacy/" className={classes.link}>
              {t('footer.privacy')}
            </Link> */}
          </Typography>
        </div>
      </div>
    </>
  )
}

export default Footer
