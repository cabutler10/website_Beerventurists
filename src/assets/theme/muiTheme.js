import { createMuiTheme } from '@material-ui/core/styles'
import 'typeface-open-sans'
import 'typeface-permanent-marker'
import './app.css'

const status = {
  white: '#ffffff',
  black: '#040001',
  greyLt: '#F7F5F5',
  greyMed: '#656565',
  grey: '#c5c6c7',
  greyDk: '#404040',
  green: '#00942b',
  yellow: '#fce566',
  red: '#FF0012',
  blue: '#015b99',
  blueLt: '#0172c0',
  blueDk: '#014472',
  orange: '#c56113',
  orangeDk: '#c56113',
  orangeLt: '#ebab2f',
  tan: '#FFFBEF',
}

const theme = createMuiTheme({
  palette: {
    type: 'light',
    primary: {
      light: status.blueLt,
      main: status.blue,
      dark: status.blueDk,
    },
    secondary: {
      light: status.orangeLt,
      main: status.orange,
      dark: status.orangeDk,
    },
    error: { main: status.red },
  },
  status,
  typography: {
    useNextVariants: true,
    fontFamily: "'Open Sans', sans-serif",
    h5: {
      letterSpacing: 3,
    },
    h2: {
      fontFamily: "'Permanent Marker', cursive",
    },
    body1: {
      letterSpacing: 1,
      lineHeight: 1.8,
      //TODO is this an error in MUI? not sure why its default is 16 when MUI default is 14
      fontSize: 14,
    },
    body2: {
      letterSpacing: 1,
      fontSize: 14,
    },
    button: {
      letterSpacing: 3,
    },
  },
  overrides: {
    MuiTypography: {
      colorPrimary: {
        color: status.black,
      },
      colorSecondary: {
        color: status.orangeLt,
      },
      colorTextSecondary: {
        color: status.white,
      },
    },
    MuiButton: {
      root: {
        padding: '9px 27px',
        borderRadius: 0,
      },
      contained: {
        boxShadow: 'none',
      },
    },
  },
})

export default theme
