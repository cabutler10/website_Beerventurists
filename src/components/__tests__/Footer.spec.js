import React from 'react'
import { render, fireEvent } from 'react-testing-library'
import Footer from '../Footer'
import { ThemeProvider } from '@material-ui/styles'
import theme from '../../assets/theme/muiTheme'
import CircularProgress from '@material-ui/core/CircularProgress'
// import { handleSubmit as mockSavePost } from '../Footer'

// jest.mock('../handleSubmit', () => {
//   return {
//     handleSubmit: jest.fn(() => Promise.resolve),
//   }
// })

// afterEach(() => {
//   mockSavePost.mockClear()
// })

describe('Footer', () => {
  const mockProps = {
    t: () => '',
  }
  it('renders correctly', () => {
    const component = render(
      <ThemeProvider theme={theme}>
        <Footer {...mockProps} />
      </ThemeProvider>
    )
    expect(component).toMatchSnapshot()
  })
  it('submits button', () => {
    const { getByTestId, getByPlaceholderText } = render(
      <ThemeProvider theme={theme}>
        <Footer {...mockProps} />
      </ThemeProvider>
    )
    const fakePost = {
      footer_name: 'mockName',
      footer_email: 'mockEmail@test.com',
      footer_message: 'mockMessage',
    }
    getByPlaceholderText('common.name').value = fakePost.footer_name
    getByPlaceholderText('common.email').value = fakePost.footer_email
    getByPlaceholderText('footer.message').value = fakePost.footer_message
    const button = getByTestId('footer_submit')
    fireEvent.click(button)
    expect(<CircularProgress />).toBeDefined()
    // expect(mockSavePost).toHaveBeenCalledTimes(1)
    // expect(mockSavePost).toHaveBeenCalledWith(fakePost)
  })
})
