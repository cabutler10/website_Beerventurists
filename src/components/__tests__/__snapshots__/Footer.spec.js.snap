// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Footer renders correctly 1`] = `
Object {
  "asFragment": [Function],
  "baseElement": <body>
    <div>
      <div
        class="Hook-root-19a211f"
      >
        <p
          class="MuiTypography-root-1 MuiTypography-body1-10 Hook-title-dyy0mi"
        >
          membership.title
        </p>
        <div>
          <img
            class="Hook-logo-10wch87"
            src="test-file-stub"
          />
          <img
            class="Hook-logo-10wch87"
            src="test-file-stub"
          />
        </div>
      </div>
      <div
        class="Hook-root-cahx9v"
      >
        <div
          class="Hook-container-j2j92c"
        >
          <div
            class="Hook-flexItem-iwwodh"
          >
            <form
              autocomplete="off"
              novalidate=""
            >
              <h5
                class="MuiTypography-root-1 MuiTypography-h5-17 Hook-textHeading-1o0r3x1"
              >
                footer.message
              </h5>
              <p
                class="MuiTypography-root-1 MuiTypography-body1-10 Hook-text-19jqmhw"
              >
                footer.questionsMessage
              </p>
              <div
                class="Hook-inputContainer-1xgvdyh"
              >
                <div
                  class="MuiFormControl-root-44 MuiFormControl-marginNormal-45 Hook-textField-135bycc"
                >
                  <label
                    class="MuiFormLabel-root-59 MuiFormLabel-required-64 MuiInputLabel-required-52 MuiInputLabel-root-48 MuiInputLabel-formControl-53 MuiInputLabel-animated-56 Hook-inputLabel-1x2xnjg"
                    data-shrink="false"
                    for="footer_name"
                  >
                    common.name
                    <span
                      class="MuiFormLabel-asterisk-65"
                    >
                       *
                    </span>
                  </label>
                  <div
                    class="MuiInputBase-root-79 MuiInput-root-66 MuiInputBase-formControl-80 MuiInput-formControl-67 Hook-input-h0iya9"
                  >
                    <input
                      aria-invalid="false"
                      class="MuiInputBase-input-89 MuiInput-input-74 Hook-textArea-e6ffmt"
                      id="footer_name"
                      placeholder="common.name"
                      required=""
                      type="text"
                      value=""
                    />
                  </div>
                </div>
                <div
                  class="MuiFormControl-root-44 MuiFormControl-marginNormal-45 Hook-textField-135bycc"
                >
                  <label
                    class="MuiFormLabel-root-59 MuiFormLabel-required-64 MuiInputLabel-required-52 MuiInputLabel-root-48 MuiInputLabel-formControl-53 MuiInputLabel-animated-56 Hook-inputLabel-1x2xnjg"
                    data-shrink="false"
                    for="footer_email"
                  >
                    common.email
                    <span
                      class="MuiFormLabel-asterisk-65"
                    >
                       *
                    </span>
                  </label>
                  <div
                    class="MuiInputBase-root-79 MuiInput-root-66 MuiInputBase-formControl-80 MuiInput-formControl-67 Hook-input-h0iya9"
                  >
                    <input
                      aria-invalid="false"
                      class="MuiInputBase-input-89 MuiInput-input-74 Hook-textArea-e6ffmt MuiInputBase-inputType-92 MuiInput-inputType-77"
                      id="footer_email"
                      placeholder="common.email"
                      required=""
                      type="email"
                      value=""
                    />
                  </div>
                </div>
              </div>
              <div
                class="MuiFormControl-root-44 MuiFormControl-marginNormal-45 MuiFormControl-fullWidth-47"
              >
                <label
                  class="MuiFormLabel-root-59 MuiFormLabel-required-64 MuiInputLabel-required-52 MuiInputLabel-root-48 MuiInputLabel-formControl-53 MuiInputLabel-animated-56 Hook-inputLabel-1x2xnjg"
                  data-shrink="false"
                  for="footer_message"
                >
                  footer.message
                  <span
                    class="MuiFormLabel-asterisk-65"
                  >
                     *
                  </span>
                </label>
                <div
                  class="MuiInputBase-root-79 MuiInput-root-66 MuiInputBase-fullWidth-88 MuiInput-fullWidth-73 MuiInputBase-formControl-80 MuiInput-formControl-67 MuiInputBase-multiline-87 MuiInput-multiline-72 Hook-input-h0iya9"
                >
                  <textarea
                    aria-invalid="false"
                    class="MuiInputBase-input-89 MuiInput-input-74 Hook-textArea-e6ffmt MuiInputBase-inputMultiline-91 MuiInput-inputMultiline-76"
                    id="footer_message"
                    placeholder="footer.message"
                    required=""
                    rows="4"
                  />
                </div>
              </div>
              <button
                class="MuiButtonBase-root-122 MuiButton-root-96 MuiButton-contained-107 MuiButton-containedPrimary-108 MuiButton-raised-110 MuiButton-raisedPrimary-111 MuiButton-fullWidth-121 Hook-buttonSend-4w8aol"
                data-testid="footer_submit"
                tabindex="0"
                type="button"
              >
                <span
                  class="MuiButton-label-97"
                >
                  footer.send
                </span>
                <span
                  class="MuiTouchRipple-root-130"
                />
              </button>
            </form>
          </div>
          <div
            class="Hook-flexItem-iwwodh"
          >
            <p
              class="MuiTypography-root-1 MuiTypography-body1-10 Hook-text-19jqmhw"
            >
              footer.getInTouch
            </p>
            <div
              class="Hook-iconContainer-181m6ap"
            >
              <a
                class="MuiButtonBase-root-122 MuiButton-root-96 MuiButton-outlined-104 Hook-button-mpjf9e"
                href="https://www.facebook.com/EisbachRiders/"
                rel="noopener"
                role="button"
                tabindex="0"
                target="_blank"
              >
                <span
                  class="MuiButton-label-97"
                >
                  <svg
                    aria-labelledby="title"
                    class="Hook-icon-11lzqnu"
                    height="32"
                    viewBox="0 0 96.124 96.123"
                    width="32"
                    xmlns="http://www.w3.org/2000/svg"
                  >
                    <title
                      id="title_facebook"
                    >
                      Facebook Icon
                    </title>
                    <path
                      d="M72.089,0.02L59.624,0C45.62,0,36.57,9.285,36.57,23.656v10.907H24.037c-1.083,0-1.96,0.878-1.96,1.961v15.803 c0,1.083,0.878,1.96,1.96,1.96h12.533v39.876c0,1.083,0.877,1.96,1.96,1.96h16.352c1.083,0,1.96-0.878,1.96-1.96V54.287h14.654 c1.083,0,1.96-0.877,1.96-1.96l0.006-15.803c0-0.52-0.207-1.018-0.574-1.386c-0.367-0.368-0.867-0.575-1.387-0.575H56.842v-9.246 c0-4.444,1.059-6.7,6.848-6.7l8.397-0.003c1.082,0,1.959-0.878,1.959-1.96V1.98C74.046,0.899,73.17,0.022,72.089,0.02z"
                      fill="currentColor"
                    />
                  </svg>
                </span>
                <span
                  class="MuiTouchRipple-root-130"
                />
              </a>
              <a
                class="MuiButtonBase-root-122 MuiButton-root-96 MuiButton-outlined-104 Hook-button-mpjf9e"
                href="https://www.instagram.com/eisbachriders/"
                rel="noopener"
                role="button"
                tabindex="0"
                target="_blank"
              >
                <span
                  class="MuiButton-label-97"
                >
                  <svg
                    aria-labelledby="title"
                    class="Hook-icon-11lzqnu"
                    height="32"
                    viewBox="0 0 448 512"
                    width="32"
                    xmlns="http://www.w3.org/2000/svg"
                  >
                    <title
                      id="title_instagram"
                    >
                      Instagram Icon
                    </title>
                    <path
                      d="M224.1 141c-63.6 0-114.9 51.3-114.9 114.9s51.3 114.9 114.9 114.9S339 319.5 339 255.9 287.7 141 224.1 141zm0 189.6c-41.1 0-74.7-33.5-74.7-74.7s33.5-74.7 74.7-74.7 74.7 33.5 74.7 74.7-33.6 74.7-74.7 74.7zm146.4-194.3c0 14.9-12 26.8-26.8 26.8-14.9 0-26.8-12-26.8-26.8s12-26.8 26.8-26.8 26.8 12 26.8 26.8zm76.1 27.2c-1.7-35.9-9.9-67.7-36.2-93.9-26.2-26.2-58-34.4-93.9-36.2-37-2.1-147.9-2.1-184.9 0-35.8 1.7-67.6 9.9-93.9 36.1s-34.4 58-36.2 93.9c-2.1 37-2.1 147.9 0 184.9 1.7 35.9 9.9 67.7 36.2 93.9s58 34.4 93.9 36.2c37 2.1 147.9 2.1 184.9 0 35.9-1.7 67.7-9.9 93.9-36.2 26.2-26.2 34.4-58 36.2-93.9 2.1-37 2.1-147.8 0-184.8zM398.8 388c-7.8 19.6-22.9 34.7-42.6 42.6-29.5 11.7-99.5 9-132.1 9s-102.7 2.6-132.1-9c-19.6-7.8-34.7-22.9-42.6-42.6-11.7-29.5-9-99.5-9-132.1s-2.6-102.7 9-132.1c7.8-19.6 22.9-34.7 42.6-42.6 29.5-11.7 99.5-9 132.1-9s102.7-2.6 132.1 9c19.6 7.8 34.7 22.9 42.6 42.6 11.7 29.5 9 99.5 9 132.1s2.7 102.7-9 132.1z"
                      fill="currentColor"
                    />
                  </svg>
                </span>
                <span
                  class="MuiTouchRipple-root-130"
                />
              </a>
            </div>
          </div>
        </div>
        <div
          class="Hook-legal-rhdqod"
        >
          <p
            class="MuiTypography-root-1 MuiTypography-body2-9 MuiTypography-alignCenter-24 Hook-copyright-1oi7zrg"
          >
            © Eisbach Riders |
             
            <a
              class="MuiTypography-root-1 MuiTypography-colorPrimary-31 MuiTypography-inline-36 MuiLink-root-125 MuiLink-underlineHover-127 Hook-link-u4zgvg"
              href="/legal/"
              style="text-decoration: none;"
            >
              footer.imprint
            </a>
            |
             
            <a
              class="MuiTypography-root-1 MuiTypography-colorPrimary-31 MuiTypography-inline-36 MuiLink-root-125 MuiLink-underlineHover-127 Hook-link-u4zgvg"
              href="/privacy/"
              style="text-decoration: none;"
            >
              footer.privacy
            </a>
          </p>
        </div>
      </div>
    </div>
  </body>,
  "container": <div>
    <div
      class="Hook-root-19a211f"
    >
      <p
        class="MuiTypography-root-1 MuiTypography-body1-10 Hook-title-dyy0mi"
      >
        membership.title
      </p>
      <div>
        <img
          class="Hook-logo-10wch87"
          src="test-file-stub"
        />
        <img
          class="Hook-logo-10wch87"
          src="test-file-stub"
        />
      </div>
    </div>
    <div
      class="Hook-root-cahx9v"
    >
      <div
        class="Hook-container-j2j92c"
      >
        <div
          class="Hook-flexItem-iwwodh"
        >
          <form
            autocomplete="off"
            novalidate=""
          >
            <h5
              class="MuiTypography-root-1 MuiTypography-h5-17 Hook-textHeading-1o0r3x1"
            >
              footer.message
            </h5>
            <p
              class="MuiTypography-root-1 MuiTypography-body1-10 Hook-text-19jqmhw"
            >
              footer.questionsMessage
            </p>
            <div
              class="Hook-inputContainer-1xgvdyh"
            >
              <div
                class="MuiFormControl-root-44 MuiFormControl-marginNormal-45 Hook-textField-135bycc"
              >
                <label
                  class="MuiFormLabel-root-59 MuiFormLabel-required-64 MuiInputLabel-required-52 MuiInputLabel-root-48 MuiInputLabel-formControl-53 MuiInputLabel-animated-56 Hook-inputLabel-1x2xnjg"
                  data-shrink="false"
                  for="footer_name"
                >
                  common.name
                  <span
                    class="MuiFormLabel-asterisk-65"
                  >
                     *
                  </span>
                </label>
                <div
                  class="MuiInputBase-root-79 MuiInput-root-66 MuiInputBase-formControl-80 MuiInput-formControl-67 Hook-input-h0iya9"
                >
                  <input
                    aria-invalid="false"
                    class="MuiInputBase-input-89 MuiInput-input-74 Hook-textArea-e6ffmt"
                    id="footer_name"
                    placeholder="common.name"
                    required=""
                    type="text"
                    value=""
                  />
                </div>
              </div>
              <div
                class="MuiFormControl-root-44 MuiFormControl-marginNormal-45 Hook-textField-135bycc"
              >
                <label
                  class="MuiFormLabel-root-59 MuiFormLabel-required-64 MuiInputLabel-required-52 MuiInputLabel-root-48 MuiInputLabel-formControl-53 MuiInputLabel-animated-56 Hook-inputLabel-1x2xnjg"
                  data-shrink="false"
                  for="footer_email"
                >
                  common.email
                  <span
                    class="MuiFormLabel-asterisk-65"
                  >
                     *
                  </span>
                </label>
                <div
                  class="MuiInputBase-root-79 MuiInput-root-66 MuiInputBase-formControl-80 MuiInput-formControl-67 Hook-input-h0iya9"
                >
                  <input
                    aria-invalid="false"
                    class="MuiInputBase-input-89 MuiInput-input-74 Hook-textArea-e6ffmt MuiInputBase-inputType-92 MuiInput-inputType-77"
                    id="footer_email"
                    placeholder="common.email"
                    required=""
                    type="email"
                    value=""
                  />
                </div>
              </div>
            </div>
            <div
              class="MuiFormControl-root-44 MuiFormControl-marginNormal-45 MuiFormControl-fullWidth-47"
            >
              <label
                class="MuiFormLabel-root-59 MuiFormLabel-required-64 MuiInputLabel-required-52 MuiInputLabel-root-48 MuiInputLabel-formControl-53 MuiInputLabel-animated-56 Hook-inputLabel-1x2xnjg"
                data-shrink="false"
                for="footer_message"
              >
                footer.message
                <span
                  class="MuiFormLabel-asterisk-65"
                >
                   *
                </span>
              </label>
              <div
                class="MuiInputBase-root-79 MuiInput-root-66 MuiInputBase-fullWidth-88 MuiInput-fullWidth-73 MuiInputBase-formControl-80 MuiInput-formControl-67 MuiInputBase-multiline-87 MuiInput-multiline-72 Hook-input-h0iya9"
              >
                <textarea
                  aria-invalid="false"
                  class="MuiInputBase-input-89 MuiInput-input-74 Hook-textArea-e6ffmt MuiInputBase-inputMultiline-91 MuiInput-inputMultiline-76"
                  id="footer_message"
                  placeholder="footer.message"
                  required=""
                  rows="4"
                />
              </div>
            </div>
            <button
              class="MuiButtonBase-root-122 MuiButton-root-96 MuiButton-contained-107 MuiButton-containedPrimary-108 MuiButton-raised-110 MuiButton-raisedPrimary-111 MuiButton-fullWidth-121 Hook-buttonSend-4w8aol"
              data-testid="footer_submit"
              tabindex="0"
              type="button"
            >
              <span
                class="MuiButton-label-97"
              >
                footer.send
              </span>
              <span
                class="MuiTouchRipple-root-130"
              />
            </button>
          </form>
        </div>
        <div
          class="Hook-flexItem-iwwodh"
        >
          <p
            class="MuiTypography-root-1 MuiTypography-body1-10 Hook-text-19jqmhw"
          >
            footer.getInTouch
          </p>
          <div
            class="Hook-iconContainer-181m6ap"
          >
            <a
              class="MuiButtonBase-root-122 MuiButton-root-96 MuiButton-outlined-104 Hook-button-mpjf9e"
              href="https://www.facebook.com/EisbachRiders/"
              rel="noopener"
              role="button"
              tabindex="0"
              target="_blank"
            >
              <span
                class="MuiButton-label-97"
              >
                <svg
                  aria-labelledby="title"
                  class="Hook-icon-11lzqnu"
                  height="32"
                  viewBox="0 0 96.124 96.123"
                  width="32"
                  xmlns="http://www.w3.org/2000/svg"
                >
                  <title
                    id="title_facebook"
                  >
                    Facebook Icon
                  </title>
                  <path
                    d="M72.089,0.02L59.624,0C45.62,0,36.57,9.285,36.57,23.656v10.907H24.037c-1.083,0-1.96,0.878-1.96,1.961v15.803 c0,1.083,0.878,1.96,1.96,1.96h12.533v39.876c0,1.083,0.877,1.96,1.96,1.96h16.352c1.083,0,1.96-0.878,1.96-1.96V54.287h14.654 c1.083,0,1.96-0.877,1.96-1.96l0.006-15.803c0-0.52-0.207-1.018-0.574-1.386c-0.367-0.368-0.867-0.575-1.387-0.575H56.842v-9.246 c0-4.444,1.059-6.7,6.848-6.7l8.397-0.003c1.082,0,1.959-0.878,1.959-1.96V1.98C74.046,0.899,73.17,0.022,72.089,0.02z"
                    fill="currentColor"
                  />
                </svg>
              </span>
              <span
                class="MuiTouchRipple-root-130"
              />
            </a>
            <a
              class="MuiButtonBase-root-122 MuiButton-root-96 MuiButton-outlined-104 Hook-button-mpjf9e"
              href="https://www.instagram.com/eisbachriders/"
              rel="noopener"
              role="button"
              tabindex="0"
              target="_blank"
            >
              <span
                class="MuiButton-label-97"
              >
                <svg
                  aria-labelledby="title"
                  class="Hook-icon-11lzqnu"
                  height="32"
                  viewBox="0 0 448 512"
                  width="32"
                  xmlns="http://www.w3.org/2000/svg"
                >
                  <title
                    id="title_instagram"
                  >
                    Instagram Icon
                  </title>
                  <path
                    d="M224.1 141c-63.6 0-114.9 51.3-114.9 114.9s51.3 114.9 114.9 114.9S339 319.5 339 255.9 287.7 141 224.1 141zm0 189.6c-41.1 0-74.7-33.5-74.7-74.7s33.5-74.7 74.7-74.7 74.7 33.5 74.7 74.7-33.6 74.7-74.7 74.7zm146.4-194.3c0 14.9-12 26.8-26.8 26.8-14.9 0-26.8-12-26.8-26.8s12-26.8 26.8-26.8 26.8 12 26.8 26.8zm76.1 27.2c-1.7-35.9-9.9-67.7-36.2-93.9-26.2-26.2-58-34.4-93.9-36.2-37-2.1-147.9-2.1-184.9 0-35.8 1.7-67.6 9.9-93.9 36.1s-34.4 58-36.2 93.9c-2.1 37-2.1 147.9 0 184.9 1.7 35.9 9.9 67.7 36.2 93.9s58 34.4 93.9 36.2c37 2.1 147.9 2.1 184.9 0 35.9-1.7 67.7-9.9 93.9-36.2 26.2-26.2 34.4-58 36.2-93.9 2.1-37 2.1-147.8 0-184.8zM398.8 388c-7.8 19.6-22.9 34.7-42.6 42.6-29.5 11.7-99.5 9-132.1 9s-102.7 2.6-132.1-9c-19.6-7.8-34.7-22.9-42.6-42.6-11.7-29.5-9-99.5-9-132.1s-2.6-102.7 9-132.1c7.8-19.6 22.9-34.7 42.6-42.6 29.5-11.7 99.5-9 132.1-9s102.7-2.6 132.1 9c19.6 7.8 34.7 22.9 42.6 42.6 11.7 29.5 9 99.5 9 132.1s2.7 102.7-9 132.1z"
                    fill="currentColor"
                  />
                </svg>
              </span>
              <span
                class="MuiTouchRipple-root-130"
              />
            </a>
          </div>
        </div>
      </div>
      <div
        class="Hook-legal-rhdqod"
      >
        <p
          class="MuiTypography-root-1 MuiTypography-body2-9 MuiTypography-alignCenter-24 Hook-copyright-1oi7zrg"
        >
          © Eisbach Riders |
           
          <a
            class="MuiTypography-root-1 MuiTypography-colorPrimary-31 MuiTypography-inline-36 MuiLink-root-125 MuiLink-underlineHover-127 Hook-link-u4zgvg"
            href="/legal/"
            style="text-decoration: none;"
          >
            footer.imprint
          </a>
          |
           
          <a
            class="MuiTypography-root-1 MuiTypography-colorPrimary-31 MuiTypography-inline-36 MuiLink-root-125 MuiLink-underlineHover-127 Hook-link-u4zgvg"
            href="/privacy/"
            style="text-decoration: none;"
          >
            footer.privacy
          </a>
        </p>
      </div>
    </div>
  </div>,
  "debug": [Function],
  "findAllByAltText": [Function],
  "findAllByDisplayValue": [Function],
  "findAllByLabelText": [Function],
  "findAllByPlaceholderText": [Function],
  "findAllByRole": [Function],
  "findAllByTestId": [Function],
  "findAllByText": [Function],
  "findAllByTitle": [Function],
  "findByAltText": [Function],
  "findByDisplayValue": [Function],
  "findByLabelText": [Function],
  "findByPlaceholderText": [Function],
  "findByRole": [Function],
  "findByTestId": [Function],
  "findByText": [Function],
  "findByTitle": [Function],
  "getAllByAltText": [Function],
  "getAllByDisplayValue": [Function],
  "getAllByLabelText": [Function],
  "getAllByPlaceholderText": [Function],
  "getAllByRole": [Function],
  "getAllBySelectText": [Function],
  "getAllByTestId": [Function],
  "getAllByText": [Function],
  "getAllByTitle": [Function],
  "getAllByValue": [Function],
  "getByAltText": [Function],
  "getByDisplayValue": [Function],
  "getByLabelText": [Function],
  "getByPlaceholderText": [Function],
  "getByRole": [Function],
  "getBySelectText": [Function],
  "getByTestId": [Function],
  "getByText": [Function],
  "getByTitle": [Function],
  "getByValue": [Function],
  "queryAllByAltText": [Function],
  "queryAllByDisplayValue": [Function],
  "queryAllByLabelText": [Function],
  "queryAllByPlaceholderText": [Function],
  "queryAllByRole": [Function],
  "queryAllBySelectText": [Function],
  "queryAllByTestId": [Function],
  "queryAllByText": [Function],
  "queryAllByTitle": [Function],
  "queryAllByValue": [Function],
  "queryByAltText": [Function],
  "queryByDisplayValue": [Function],
  "queryByLabelText": [Function],
  "queryByPlaceholderText": [Function],
  "queryByRole": [Function],
  "queryBySelectText": [Function],
  "queryByTestId": [Function],
  "queryByText": [Function],
  "queryByTitle": [Function],
  "queryByValue": [Function],
  "rerender": [Function],
  "unmount": [Function],
}
`;
